crates/hyperswitch_connectors/src/connectors/taxjar.rs

```diff
@@ -190,11 +190,22 @@ impl ConnectorIntegration<CalculateTax, PaymentsTaxCalculationData, TaxCalculati
 
         let shipping = utils::convert_amount(
             self.amount_converter,
-            req.request.shipping_cost.unwrap_or(MinorUnit::new(0)),
+            req.request.shipping_cost.unwrap_or(MinorUnit::zero()),
             req.request.currency,
         )?;
 
-        let connector_router_data = taxjar::TaxjarRouterData::from((amount, shipping, req));
+        let order_amount = utils::convert_amount(
+            self.amount_converter,
+            req.request
+                .order_details
+                .as_ref()
+                .and_then(|details| details.first().map(|item| item.amount))
```

Comment:
Aren't we assuming we only have one item here? Should we consider summing amounts of all items instead?